ckanext/textview/theme/public/text_view.js

```diff
@@ -58,7 +58,9 @@ ckan.module('text_view', function (jQuery) {
           if (p.language) {
             highlighted = hljs.highlight(p.language, data, true).value;
           } else {
-            highlighted = '<pre>' + data + '</pre>';
+            var escape_ = document.createElement('textarea');
+            escape_.textContent = data;
+            highlighted = '<pre>' + escape_.innerHTML + '</pre>';
```

Comment:
```suggestion
            highlighted = $('<pre></pre>').text(data)[0].outerHTML;
```
Textarea is not really needed, any element would do. Instead of creating new elements I would just create the pre element, insert the data as text and return outerHTML to keep the functionality as the same as before.